src/audio.cpp

```diff
@@ -153,14 +153,16 @@ namespace audio {
     }
 
     // Order of priority:
-    // 1. Config
-    // 2. Virtual if available
+    // 1. Virtual sink
```

Comment:
Wouldn't the config option still be the first priority?

Reply:
It was somewhat confusing as written before because Virtual Sink and Audio Sink are both "Config". This clarifies the prescedence between the two config options.